### DISCUSSION TITLE
Better compatibility with Aerospace auto tiling

### DISCUSSION THREAD

**Discussion by martinengelke** [lawand-dot-io/taskbar#117] [Feature requests]
**Describe the bug**
Auto resize is not working well with Aerospace auto tiling 
Seems like taskbar tries to force auto size and Aerospace starts to resize immediately which causes flickering.

**To Reproduce**
Download an use https://github.com/nikitabobko/AeroSpace
Enable auto-resize windows

**Expected behavior**
Taskbar should rather act like an overlap then an window itself, which would then prevent tiling managers recognise them as windows itself.

if your bundleID is really:

com.fpfxtknjju.wbgcdolfev

I added it to the tiling manager for now to prevent the issue.

You can see an implementation of an overlapping tool right here:
https://www.apptorium.com/sidenotes
which did not conflict anything for month now.

**Desktop (please complete the following information):**
 - Taskbar version latest
 - OS: 14.5 (23F79)
 - Chip M3Pro

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the bug report. I might not be able to do anything about this as I can't control how Taskbar interacts with all third party apps, and rather only with macOS itself. I will try to download this app and see if there's anything I can do about this. Cheers

**Comment by lawand-dot-io** [comment]
Will convert this into a feature request. Cheers
